config/application.rb

```diff
@@ -20,6 +20,7 @@ class Application < Rails::Application
       g.test_framework = :rspec
     end
 
+    config.autoload_paths += %W(#{config.root}/lib)
```

Comment:
Can you remind me why you added this piece?

Reply:
I tried running `rake db:seed` with that `autoload_paths` code commented out and it still worked. Let me know what this line adds.